functions/auth.js

```diff
@@ -1,5 +1,16 @@
 const functions = require("firebase-functions");
+const admin = require("firebase-admin");
+
+const { HttpsError } = require("firebase-functions/lib/providers/https");
+
+const { fetchAuthConfig } = require("./src/api/auth");
+
+const { createOAuth2Client } = require("./src/utils/auth");
+const { getJson, postJson } = require("./src/utils/fetch");
+const { checkIfValidVenueId } = require("./src/utils/venue");
+
 const PROJECT_ID = functions.config().project.id;
+const { origin: AUTH_ORIGIN } = functions.config().auth;
```

Comment:
[TODO] Do we want to keep this here, or move it into the authConfig in firestore? I think authConfig. And maybe name it better. It's essentially used as part of the 'redirect back to the platform' code.

Reply:
See https://github.com/sparkletown/sparkle/pull/1523#discussion_r653436631 for why this might need to stay as is.